service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -146,6 +158,20 @@ void aboutToSubmitShouldAddC21OrderToCaseDataAndRemoveTemporaryCaseDataOrderFiel
             aboutToSubmitAssertions(callbackResponse.getData(), expectedC21Order);
         }
 
+        @Test
+        void aboutToSubmitShouldNotHaveDraftAppendedToFilename() {
+            final CaseDetails caseDetails = buildCaseDetails(
+                commonCaseDetailsComponents(CARE_ORDER, FINAL)
```

Comment:
commonCaseDetailsComponents 🤔  what is this?

Reply:
this method already existed in the class and I just used it, it builds order type and document, legal advisor, family man case number - some common things needed among all the methods for case details then more is added upon after that... happy to change method name if you would like?